### ISSUE TITLE
sc.pl.paga: TypeError: paga() got an unexpected keyword argument 'ncols'

### ISSUE THREAD

**Issue by YubinXie** [scverse/scanpy#1203]
The current paga plot output a n_row =1 plot, which is not very user-friendly. It would be nice to have an option to control n_col as other plot functions in scanpy do. Thanks!

**Comment by flying-sheep** [comment]
I’m focusing on the new holoviz plotting now, but I’ll happily merge a PR if someone makes one

**Comment by micheleoliveiracod** [comment]
Hi, I'd like to work on this if it's still open.

Currently sc.pl.paga() always renders as a single row, unlike other plotting functions in scanpy that already expose an ncols parameter for grid layout. I'd propose adding an ncols argument to sc.pl.paga() following the same convention used elsewhere in the plotting module, so the multi-panel output can be arranged in a grid instead of one long row.

Before starting: is there a specific existing function (e.g. in sc.pl) I should mirror for the parameter's default value and validation, or is any consistent implementation acceptable here?
